TradeItIosTicketSDK2/NumberFormatter.swift

```diff
@@ -1,29 +1,40 @@
 class NumberFormatter: NSObject {
-    private static let formatter = Foundation.NumberFormatter()
+    private static let currencyFormatter = Foundation.NumberFormatter()
+    private static let quantityFormatter = Foundation.NumberFormatter()
+    private static let percentageFormatter = Foundation.NumberFormatter()
     
-    static func formatCurrency(_ number: NSNumber, maximumFractionDigits: Int = 2, currencyCode: String?) -> String {
+    static func formatCurrency(_ number: NSNumber, maximumFractionDigits: Int = 2, variance: Bool = false, currencyCode: String?) -> String {
```

Comment:
- `variance` -> `displayVariance`
- Think about splitting formatting and presentation into the appropriate formatter/presenter.